products/appengine/terraform.yaml

```diff
@@ -107,7 +107,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "web_service"
         vars:
           bucket_name: "appengine-test-bucket"
-
+  ServiceSplitTraffic: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{service_id}}"
+    import_format: ["apps/{{project}}/services/{{service_id}}"]
+    mutex: "apps/{{project}}/services/{{service_id}}"
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      custom_delete: templates/terraform/custom_delete/skip_delete.go.erb
+      test_check_destroy: templates/terraform/custom_check_destroy/skip_delete_during_test.go.erb
+    properties:
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: 'service_id'
+        ignore_read: true  
+      split: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true    
+      split.allocations: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_flatten: templates/terraform/custom_flatten/float64_to_string.go.erb
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "app_engine_service_split_traffic"
+        vars:
+          service_id: "default"
+          split.allocations.v1: "1"
+          bucket_name: "appengine-static-content"
+        test_env_vars:
+          org_id: :ORG_ID    
+        skip_test: true  
```

Comment:
Do you remember why we had to skip adding a test? I think we can generate one.

Reply:
That's the problem, if I do not skip_test and generate it I get the error which I fixed using the work around. I have done quite a bit of debugging but could not reach to the bottom of this issue.

Reply:
Would you mind pushing a commit with the test enabled so I can take a look again? I'd really like to get it working.